src/styles/global.scss/style.scss

```diff
@@ -1,38 +1,41 @@
+@import 'global.scss/variables';
+@import 'global.scss/mixins';
+
 body {
```

Comment:
Put styles for tags in `global.scss` **file**
Firstly, rename **folder**: `global.scss` -> `global`
And after create file `global.scss `